src/protocol/codecs/topic.rs

```diff
@@ -81,6 +107,24 @@ impl TopicCodec {
 
         Ok(topics)
     }
+
+    /// Marshall topics to a byte stream.
+    fn marshall_topics(&mut self, topics: Vec<Topic>) -> BytesMut {
```

Comment:
We are aligning the function name to the original implementation here
https://github.com/algorand/go-algorand/blob/8a335bb2b10ea3cb6e9471526701bcaa4089b53d/network/msgOfInterest.go#L33